### ISSUE TITLE
Installing beta/nightly leaves npm horked

### ISSUE THREAD

**Issue by doug-wade** [tj/n#378]
I used the method suggested in https://github.com/tj/n/issues/376 to install beta 3, and I discovered that npm is pretty horked when you install a beta build

```
doug-UX301LAA ~ » NODE_MIRROR=https://nodejs.org/download/test/ n v7.0.0-test2016100609987d242b
doug-UX301LAA ~ » npm i -g sinopia
module.js:475
    throw err;
    ^

Error: Cannot find module 'internal/fs'
    at Function.Module._resolveFilename (module.js:473:15)
    at Function.Module._load (module.js:421:25)
    at Module.require (module.js:501:17)
    at require (internal/module.js:20:19)
    at evalmachine.<anonymous>:17:20
    at Object.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-registry-client/node_modules/graceful-fs/fs.js:11:1)
    at Module._compile (module.js:574:32)
    at Object.Module._extensions..js (module.js:583:10)
    at Module.load (module.js:491:32)
    at tryModuleLoad (module.js:450:12)
doug-UX301LAA ~ » node -v                                                                                                                                                                                                         1 ↵
v7.0.0-test2016100609987d242b
doug-UX301LAA ~ » npm -v
module.js:475
    throw err;
    ^

Error: Cannot find module 'internal/fs'
    at Function.Module._resolveFilename (module.js:473:15)
    at Function.Module._load (module.js:421:25)
    at Module.require (module.js:501:17)
    at require (internal/module.js:20:19)
    at evalmachine.<anonymous>:17:20
    at Object.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-registry-client/node_modules/graceful-fs/fs.js:11:1)
    at Module._compile (module.js:574:32)
    at Object.Module._extensions..js (module.js:583:10)
    at Module.load (module.js:491:32)
    at tryModuleLoad (module.js:450:12)
doug-UX301LAA ~ » uname -a                                                                                                                                                                                                        1 ↵
Linux doug-UX301LAA 3.13.0-24-generic #47-Ubuntu SMP Fri May 2 23:30:00 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
```

I'm fairly certain this is an `n` issue, rather than a node 7 issue, since this repros for a nightly build

```
doug-UX301LAA ~/node-v7.0.0-test2016100609987d242b-linux-x64 » NODE_MIRROR=https://nodejs.org/download/nightly/ n 7.0.0-nightly20160930f5d997c476

     install : node-v7.0.0-nightly20160930f5d997c476
       mkdir : /usr/local/n/versions/node/7.0.0-nightly20160930f5d997c476
       fetch : https://nodejs.org/download/nightly/v7.0.0-nightly20160930f5d997c476/node-v7.0.0-nightly20160930f5d997c476-linux-x64.tar.gz
######################################################################## 100.0%
   installed : v7.0.0-nightly20160930f5d997c476

doug-UX301LAA ~/node-v7.0.0-test2016100609987d242b-linux-x64 » npm -v
module.js:475
    throw err;
    ^

Error: Cannot find module 'internal/fs'
    at Function.Module._resolveFilename (module.js:473:15)
    at Function.Module._load (module.js:421:25)
    at Module.require (module.js:501:17)
    at require (internal/module.js:20:19)
    at evalmachine.<anonymous>:17:20
    at Object.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-registry-client/node_modules/graceful-fs/fs.js:11:1)
    at Module._compile (module.js:574:32)
    at Object.Module._extensions..js (module.js:583:10)
    at Module.load (module.js:491:32)
    at tryModuleLoad (module.js:450:12)
```

**Comment by fabiancook** [comment]
Related https://github.com/npm/npm/issues/14232 

Solves issue by installing npm@next beforehand 

```
n latest
npm i -g npm@next
NODE_MIRROR=https://nodejs.org/download/test/ n v7.0.0-test2016100609987d242b
npm i 
```

**Comment by JohnRGee** [comment]
This was probably fixed by #470
